sdk/include/opentelemetry/sdk/metrics/aggregation/default_aggregation.h

```diff
@@ -90,6 +91,53 @@ class DefaultAggregation
         return DefaultAggregation::CreateAggregation(instrument_descriptor);
     }
   }
+
+  static std::unique_ptr<Aggregation> CloneAggregation(AggregationType aggregation_type,
+                                                       InstrumentDescriptor instrument_descriptor,
+                                                       const Aggregation &to_copy)
+  {
+    const PointType point_data = to_copy.ToPoint();
+    switch (aggregation_type)
+    {
+      case AggregationType::kDrop:
+        return std::unique_ptr<Aggregation>(new DropAggregation());
```

Comment:
Probably make assignment of the new Aggregation in the switch and return it at the end of the function? `return` on each case could cause multiple copy of epilog being generated.

Reply:
you mean multiple stack unwinding code being generated. I think modern compilers should be smart enough to handle that.